parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -25,6 +25,10 @@
 import java.io.Serializable;
 import java.io.UnsupportedEncodingException;
 import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
```

Comment:
Yeah, this is actually PARQUET-415. I've included that commit just to make the tests pass. It's been reviewed so I'll merge it and then update the PR to exclude it.